unity/Hello_World/ProjectSettings/ProjectSettings.asset

```diff
@@ -193,6 +193,7 @@ PlayerSettings:
   - {fileID: 0}
   - {fileID: 0}
   - {fileID: 0}
+  - {fileID: 0}
```

Comment:
This doesn't seem related to anything? I don't know why changes like this happen or what they mean.